<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Corsa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <disabledoff>../../logo.png</disabledoff>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 52, 52);</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Mezzi disponibili:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_inizia_corsa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>▶ Inizia
  Corsa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_termina_corsa">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>⏹ Termina
  Corsa</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>Torna Indietro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
